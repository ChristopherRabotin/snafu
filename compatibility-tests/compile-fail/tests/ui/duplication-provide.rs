use snafu::prelude::*;

#[derive(Debug, Snafu)]
struct Error {
    #[snafu(provide)]
    #[snafu(provide)]
    name: String,
}

fn main() {}
