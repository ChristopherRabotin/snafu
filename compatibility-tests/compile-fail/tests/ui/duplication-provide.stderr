error: Multiple `provide` attributes are not supported on a field
 --> tests/ui/duplication-provide.rs:6:13
  |
6 |     #[snafu(provide)]
  |             ^^^^^^^
